docs/guide/content_rendering/design_engine/add_new_design.md

```diff
@@ -0,0 +1,108 @@
+# Add new design
+
+To create different designs for different version of the website,
+you configure different sites based on the [SiteAccess](../../multisite/multisite.md) concent.
+
+This example shows how to prepare a site for a "Summer Sale" marketing campaign
+and provide it with a distinct design.
+
+## Configure a new SiteAccess
+
+First, in the SiteAccess configuration in `config/packages/ezplatform.yaml`,
+add the `campaign` SiteAccess to the list under `ezplatform.siteaccess`:
+
+``` yaml
+ezplatform:
+    siteaccess:
+        list: [site, campaign]
+        groups:
+            site_group: [site, campaign]
+        default_siteaccess: site
+```
+
+Adding the `campaign` SiteAccess to [`site_group`](../../multisite/multisite_configuration.md#siteaccess-groups) enables you to add common configuration for both SiteAccesses at the same time.
+
+!!! tip
+
+    For details about configuring different site roots and matching SiteAccesses, see [Set up campaign SiteAccess](../../multisite/setup_campaign_siteaccess.md).
+
+## Create themes
+
+Next, configure a new `summersale` design this a theme also named `summersale`:
```

Comment:
I think some part of this sentence is missing?

Reply:
```suggestion
Next, configure a new `summersale` design for this theme, also named `summersale`:
```